loops_lab.js

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
I think you understand how to use/create loops to solve problems, and also have good understanding of how loops work, but dealing with logics inside loops remains matter of further practice. Please read questions carefully next time, and also please reach out to me or Alejo if you have any questions or clarifications. It is important to get good basic understanding of Javascript in early stage. Keep practicing and keep up the good work!